.github/workflows/create-quantic-package.yml

```diff
@@ -46,20 +46,14 @@ jobs:
               repo: context.repo.repo,
               run_id: context.runId
             })
-      - name: Install NPM dependencies
-        working-directory: ./packages/quantic
-        run: |
-          npm i -g typescript
-          npm i -g sfdx-cli
-          npm i
+      - name: Setup
+        uses: ./.github/actions/setup
       - name: Build Quantic project
         working-directory: ./packages/quantic
         run: npm run build
```

Comment:
You should no longer need the build step, since it's included in the `setup` action

Reply:
Ah yes perfect